e3db/client.py

```diff
@@ -329,9 +329,9 @@ def __delete_access_key(self, writer_id, user_id, reader_id, record_type):
         response = requests.delete(url=url, auth=self.e3db_auth)
         self.__response_check(response)
 
-        ak_cache_key = (writer_id, user_id, record_type)
-        if ak_cache_key in self.ak_cache:
-            del self.ak_cache[ak_cache_key]
+        # ak_cache_key = (str(writer_id), str(user_id), record_type)
```

Comment:
commented initially because I wasn't sure why we'd ever want to delete our own access keys, but since they can be rolled it makes sense -- removing

Reply:
I'm actually not convinced this is a good idea. When we add support for rolling keys, we'll need to adjust how the cache works anyway. Probably re-index by ID or some such. But this still just seems to make things less efficient to me.

Reply:
For posterity: re-removed this change __delete_access_key no longer invalidates writer cache